flake8_builtins.py

```diff
@@ -64,16 +69,19 @@ def run(self):
                     yield line, offset, msg, rtype
 
     def check_assignment(self, statement):
+        is_class_def = type(statement.__flake8_builtins_parent) is ast.ClassDef
```

Comment:
I very much prefer to have a regular ``if`` conditional, these inline boolean expressions are easy to oversee while refactoring.

Something like:
```python
var = False
if condition:
  var = True
```